packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -168,6 +172,27 @@ function createResourceTimingSpans(entries, filterUrls) {
   return spans
 }
 
+function createUserTimingSpans(entries) {
+  const userTimingSpans = []
+
+  for (let i = 0; i < entries.length; i++) {
+    const { name, startTime, duration } = entries[i]
+
+    if (duration <= USER_TIMING_THRESHOLD) {
+      continue
+    }
+    const kind = 'user.measure'
+    const end = startTime + duration
+    const span = new Span(name, kind)
+    span._start = startTime
+    span.ended = true
```

Comment:
Maybe call the `span.end` before this to make sure any changes in `span.end` would be reflected here as well.